### ISSUE TITLE
[bug?] Can't specify drop_last argument in  Datasets.dataloaders()

### ISSUE THREAD

**Issue by richardyy1188** [fastai/fastai2#165]
In `35_tutorial.wikitext.ipynb`, I call `dsets.dataloaders(bs=bs, seq_len=seq_len,drop_last=True)`, and get
`TypeError: type object got multiple values for keyword argument 'drop_last'`  
  
This is because  in `fastai2/data/core.py`  
```
class FilteredBase:
....
    def dataloaders(self, bs=64, val_bs=None, shuffle_train=True, n=None, path='.', dl_type=None, dl_kwargs=None, device=None,
                  **kwargs):
....
        dl = dl_type(self.subset(0), bs=bs, shuffle=shuffle_train, drop_last=shuffle_train, n=n, device=device,
                     **merge(kwargs, dl_kwargs[0]))
```
`drop_last` has already been specified as `shuffle_train`. I can't get the reason that whether to shuffle the trainset relates to whether to drop the last batch, so I guess it is a bug.

**Comment by sgugger** [comment]
You can pass it now, but note that it doesn't really change the existing behavior: `shuffle_train` with set it to `True` and the validation dataloader will always set it to False.

**Comment by richardyy1188** [comment]
Is there a reason that we can't drop the last batch of validataion dataset ?
      
When I set `seq_len=200`, I get last batch of shape `(batch_size, 133)` which breaks my model.  
  
And my model is kind of like this, so it is hard to adapt to batch with different sequence length.
```
class ExperimentTextModel(...):
  def __init__(...):
    ...
    self.norm = nn.BatchNorm1d(config.sequence_length)
    ...
```
Or can I get `(bs, 200)` for the last batch which `bs` is smaller than `batch_size` I specified?
